.eslintrc

```diff
@@ -0,0 +1,35 @@
+env:
+  node: true
+  browser: true
+
+rules:
+  brace-style: 2
+  comma-spacing: 1
+  eol-last: 0
+  indent: [2, 2]
+  key-spacing: 0
+  new-parens: 2
+  no-debugger: 2
+  no-dupe-args: 2
+  no-dupe-keys: 2
+  no-duplicate-case: 2
+  no-ex-assign: 2
+  no-extra-parens: 1
+  no-fallthrough: 2
+  no-multi-spaces: [2, { exceptions: { "ObjectExpression": true } }]
```

Comment:
that is actually [your code](https://github.com/volumio/Volumio2/commit/2414e471e3bb024d7f822727955b5afc5a051415#diff-3c6db640873e19d10724061e8ba731e8R57) :stuck_out_tongue_winking_eye: